addons/account_edi_ubl_cii/models/account_edi_xml_ubl_20.py

```diff
@@ -318,6 +317,9 @@ def _get_invoice_line_allowance_vals_list(self, line, tax_values_list=None):
             # https://docs.peppol.eu/poacc/billing/3.0/codelist/UNCL5189/
             'allowance_charge_reason_code': 95,
 
+            'multiplier_factor': line.discount / 100.0,
+            'base_amount': gross_price_subtotal,
```

Comment:
Also, I wouldn't add the `base_amount` by default, because it complexify the line calculation although it's optional. You can keep it inside the template, but let's not use it by default.